bliss/conf/base_config.yaml

```diff
@@ -102,26 +105,26 @@ encoder:
         head: [
             [-1, 1, Conv, [512, 1, 1]],
             [-1, 1, nn.Upsample, [None, 2, 'nearest']],
-            [[-1, 6], 1, Concat, [1]],
+            [[-1, 8], 1, Concat, [1]],
             [-1, 3, C3, [512, false]],
             [-1, 1, Conv, [256, 1, 1]],
             [-1, 1, nn.Upsample, [None, 2, 'nearest']],
-            [[-1, 4, 5], 1, Concat, [1]],
+            [[-1, 6, 7], 1, Concat, [1]],
             [-1, 3, C3, [256, false]],
-            [[17], 1, Detect, [nc, anchors]],
+            [[-1], 1, Detect, [nc, anchors]],
         ]
 
 generate:
     n_workers_per_process: 0
     n_batches: ${simulator.n_batches}
     batch_size: ${simulator.prior.batch_size}
-    max_images_per_file: 4096
+    max_images_per_file: 2048
```

Comment:
better to leave it as 4096, unless there's a reason related to this PR to change it